app/docker/views/containers/create/createContainerController.js

```diff
@@ -202,19 +202,18 @@ function ($q, $scope, $async, $state, $timeout, $transition$, $filter, Container
     }
     config.HostConfig.NetworkMode = networkMode;
     config.MacAddress = $scope.formValues.MacAddress;
-
-    let aliases = [];
-    if (networkMode && _.has($scope.config.NetworkingConfig.EndpointsConfig[networkMode], 'Aliases')){
-      aliases = _.without($scope.config.NetworkingConfig.EndpointsConfig[networkMode].Aliases, $scope.config.Hostname);
-    }
-    
+ 
     config.NetworkingConfig.EndpointsConfig[networkMode] = {
       IPAMConfig: {
         IPv4Address: $scope.formValues.IPv4,
         IPv6Address: $scope.formValues.IPv6
-      },
-      Aliases: aliases
+      }
     };
+    
+    if (networkMode && _.get($scope.config.NetworkingConfig.EndpointsConfig[networkMode], 'Aliases') != null){
```

Comment:
Any particular reason for using this condition instead of the original one?

```
if (networkMode && _.has($scope.config.NetworkingConfig.EndpointsConfig[networkMode], 'Aliases')){
}
```

Reply:
I was aiming to not set aliases as `[]` when originally `null`

Reply:
`_.has()` returns `true` when path exists (so if `.Aliases` is `null` it returns `true`)
`_.get` returns `null` or `<value>`